.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
on another line, please add 
```
node_modules/
```
this should stop all of your node modules from being uploaded next time :-)